managers/asset-operations-manager.js

```diff
@@ -355,11 +356,15 @@ class AssetOperationsManager {
             if (unfinalizedState != null && unfinalizedState !== ZeroHash) {
                 publicAssertionId = unfinalizedState;
             }
+
+            stateFinalized = false;
```

Comment:
With the LATEST option, state can be finalized or unfinalized, depends if there is a pending state and we're not checking it here